doc/classes/Control.xml

```diff
@@ -1372,7 +1372,7 @@
 			Automatic layout direction, determined from the parent control layout direction.
 		</constant>
 		<constant name="LAYOUT_DIRECTION_APPLICATION_LOCALE" value="1" enum="LayoutDirection">
-			Automatic layout direction, determined from the current locale.
+			Automatic layout direction, determined from the current locale. Right-to-left layout direction is automatically used for languages that require it such as Arabic and Hebrew., but only if a valid translation file is loaded for the given language. For all other languages (or if no valid translation file is found by Godot), left-to-right layout direction is used. If using [TextServerFallback] ([member ProjectSettings.internationalization/rendering/text_driver]), left-to-right layout direction is always used regardless of the language.
```

Comment:
> but only if a valid translation file is loaded for the given language

After https://github.com/godotengine/godot/pull/102649, it's also possible to set `internationalization/locale/fallback` to RTL language without translation files (if it's the only supported language).

There's also `internationalization/rendering/force_right_to_left_layout_direction` property for testing.

Reply:
Oops I overlooked this when merging. That can be improved in a follow-up.